src/hooks/useMultiColumnListSort.test.js

```diff
@@ -0,0 +1,62 @@
+import {
+  renderHook,
+  act,
+} from '@testing-library/react-hooks';
+
+import useMultiColumnListSort from './useMultiColumnListSort';
+
+import { sortOrders } from '../constants';
+
+describe('Given useMultiColumnListSort', () => {
+  it('should return default values', () => {
+    const currentParameters = {
+      sortOrder: '',
+      sortedColumn: '',
+    };
+    const { result } = renderHook(() => useMultiColumnListSort(currentParameters.sortOrder, currentParameters.sortedColumn));
+    const [sortParameters] = result.current;
+
+    expect(sortParameters.sortOrder).toBe('');
+    expect(sortParameters.sortedColumn).toBe('');
+  });
+
+  describe('when handle "onChange" action', () => {
+    it('should return ascending sortOrder and selected sortedColumn', () => {
+      const currentParameters = {
+        sortOrder: {
+          name: 'asc',
+          fullName: 'ascending',
+        },
+        sortedColumn: 'newName',
+      };
+      const { result } = renderHook(() => useMultiColumnListSort(currentParameters.sortOrder, currentParameters.sortedColumn));
+      const [sortParameters, onChange] = result.current;
+
+      act(() => {
+        onChange('', { name: 'newName' });
+      });
+
+      expect(sortParameters.sortOrder).toEqual(sortOrders.asc);
+      expect(sortParameters.sortedColumn).toBe('newName');
+    });
+
+    it('should return descending sortOrder and selected sortedColumn', () => {
```

Comment:
There are two cases we should check:
1. `onChange` is called with the same `name` as `currentColumn`. Then `sortOrder` should switch from `asc` to `desc` or vice versa
2. `onChange` is called with a different `name` as `currentColumn`. Then `sortOrder` should be set to default value `asc` and `currentColumn` should be equal to `name`

This test is equivalent to the previous one - they both check the first case. Please change this test to call `onChange` with a different name and check that correct default `sortOrder` is set